### PR TITLE
Resolved issue #6 navbar added and improper alignment of the sections resolved

### PR DESCRIPTION
**Pull request by singhrimiumesh**
**Changes Made**

Added a navbar to the page for better navigation.
Adjusted alignment of the sections to ensure proper layout.

**Details**

Added HTML structure for the navbar and included necessary CSS styles.
The sticky navbar enhances user experience by providing easy access to navigation options.
Ensured that the navbar spans the full width of the page for improved aesthetics.
Corrected alignment issues with the sections to maintain a consistent layout across different screen sizes.

**Screenshots**

Before : no navbar and improper alignment
Screenshot 2024-05-11 123709

Now  : Added Sticky Navbar and resolved the alignment issue
Screenshot 2024-05-11 123728

### CODE REVIEW

**Comment by singhrimiumesh** [comment]
@Its-Aman-Yadav , and you please review and add the  label as gssoc24 and level1

**Comment by Its-Aman-Yadav** [comment]
Navbar is not looking so good make it more clear and visible @singhrimiumesh

File: css/style.css [hunk 1]
```diff
@@ -71,6 +77,65 @@ strong {
   font-weight: 700;
 }
 
+
+nav {
+  background-color: rgba(255, 255, 255, 0.2);
```
Comment (Its-Aman-Yadav) [inline css/style.css:82]: Please make it more clear

**Review by Its-Aman-Yadav — APPROVED** [review]

**Comment by bhaavvya** [comment]
Heyy @singhrimiumesh can you please resolve the issues.

**Comment by Its-Aman-Yadav** [comment]
Hey @singhrimiumesh are you still working on it?

**Comment by Its-Aman-Yadav** [comment]
image
It is looking like this, please make changes

**Review by Its-Aman-Yadav — CHANGES_REQUESTED** [review]
remove the  old navbar and place yours

**Pull request closed without merging**